src/features/bookings/components/TicketBody/TicketBody.tsx

```diff
@@ -0,0 +1,48 @@
+import { addDays, isSameDay } from 'date-fns'
+import React, { FunctionComponent } from 'react'
+
+import { BookingReponse, SubcategoryIdEnum, WithdrawalTypeEnum } from 'api/gen'
+import { DefaultBody } from 'features/bookings/components/TicketBody/DefaultBody/DefaultBody'
+import { EmailSent } from 'features/bookings/components/TicketBody/EmailSent/EmailSent'
+import { NoTicket } from 'features/bookings/components/TicketBody/NoTicket/NoTicket'
+import { QrCode } from 'features/bookings/components/TicketBody/QrCode/QrCode'
+
+type Props = {
+  booking: BookingReponse
```

Comment:
je pense que ça serait mieux de passer uniquement les data nécessaires à ce composant

```suggestion
  withdrawalDelay?: number
  withdrawalDetails?: string
  withdrawalType?: WithdrawalTypeEnum // j'hésite avec withdrawalType: WithdrawalTypeEnum
  subcategoryId: SubcategoryIdEnum
  beginningDatetime?: string
  qrCodeData?: string
```

ça simplifirait les tests et/ou les stories